tools/c-parser/CTranslation.thy

```diff
@@ -180,4 +180,7 @@ lemma proc_deps_lvar_nondet_init [simp]:
 
 declare word_neq_0_conv[simp] (* FIXME: for backward compatibility; remove *)
 
+(* Remove separation logic notation, because it leads to too many syntax ambiguities in general *)
+unbundle no C_seplog_syntax
```

Comment:
not clear on why you need this when I don't see you having unbundled C_seplog_syntax outside of Separation.thy ... is this so that when people include Separation.thy they don't need to additionally unbundle it? CTranslation includes something that unbundles it? Feels like it's not a commonly used feature

Reply:
CTranslation.thy (transitively) includes Separation.thy, that's how the syntax became global visible in the first place. This one just switches off the bundle from Separation.thy again. Will add that to the comment.